boards/jiminy-mega256rfr2/Makefile.include

```diff
@@ -0,0 +1,17 @@
+include $(RIOTBOARD)/common/arduino-atmega/Makefile.include
+
+# define the cpu used by the arduino uno board
+export CPU = atmega256rfr2
+
+USEMODLUE += periph
+
+# export needed for flash rule
+export PORT_LINUX ?= /dev/ttyACM0
+export PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+export PROGRAMMER_SPEED ?= 0010005
```

Comment:
Could we not sue B500000 instead of 0010005?

please add comment on where to find supported baudrates
```
# Serial Baud rate is configured to 500kBaud
# see include/asm-generic/termbits.h for availabel baudrates on your linux system
```

Reply:
No, B500000 is defined in termbits.h. When adding it to the Makefile it wouldn't get resolved preprocessor and AVRDUDE will complain about an invalid command